CI/unit_tests/core/test_data_classes.py

```diff
@@ -0,0 +1,138 @@
+"""
```

Comment:
It is sort of up to you but writing doc strings under your tests helps people know exactly what the test is doing and what should happen in it.

Reply:
In the case of tests it is often
- sufficient to look at the name of the test to understand what it tests.
- easy to understand what the tests are doing, when looking at the short code.
- confusing to add docstrings for a method `test_dvc_params_affected_files` that tests `dvc.affected_files` in 4 lines of code.
- not necessary to have docstrings because tests, once written don't need any attention by the user.

Therefore, I agree that long and complex tests, such as integrations tests can benefit from docstrings, but the majority of tests should be short tests that are mostly self explanatory and don't require docstrings, because they don't contain any additional information and will not be displayed in the documentation anyway.

Reply:
Yes but there is also a concept in coding of independent of how well you wrote your code or named your variables, it is never a reason to not have documentation. 

At the same time, I did not mention it for my benefit, only as an observation, so as long as you are comfortable with it, that is okay.